# Ecología Teórica Posgrado
Notas, lecturas para un curso de Ecología Teórica a nivel posgrado




----
## 2023


### Artículos

#### Time-series and models

- Lana Descheemaeker Sophie de Buyl (2020) Stochastic logistic models reproduce experimental time series of microbial communities eLife 9:e55650.
https://doi.org/10.7554/eLife.55650
- Ho, P. Y., Good, B. H., & Huang, K. C. (2022). Competition for fluctuating resources reproduces statistics of species abundance over time across wide-ranging microbiotas. Elife, 11, e75168.
- Sadhu, S., & Thakur, S. C. (2022). Analysis of bistable behavior and early warning signals of extinction in a class of predator-prey models. arXiv preprint arXiv:2210.04097.
- Calleja-Solanas, V., Khalil, N., Gómez-Gardeñes, J., Hernández-García, E., & Meloni, S. (2022). Structured interactions as a stabilizing mechanism for competitive ecological communities. Physical Review E, 106(6), 064307.
- Emergent ecosystem functions follow simple quantitative rules Diaz-Colunga, J.*, Skwara, A*., Vila, J.C.C., Bajic, D. & Sanchez A. BioRxiv (2022) https://www.biorxiv.org/content/10.1101/2022.06.21.496987v1.full.pdf
- Emergent coexistence in multispecies microbial communities Chang, C-Y., Bajic, D., Vila, J.C.C., Estrela, S. & Sanchez A. BioRxiv (2022) https://www.biorxiv.org/content/10.1101/2022.05.20.492860v2.full.pdf
-  Diversity begets diversity under metabolic niche construction.
Estrela S., Diaz-Colunga, J., Vila, J.C.C., Sanchez-Gorostiaga, A., Sanchez, A.
eLife (2023) [BioRxiv (2023)https://www.biorxiv.org/content/10.1101/2022.02.13.480281v1.full.pdf

- [Modelos estocásticos para estudiar la ecología del microbioma](https://youtu.be/hginU4o9vmg) Román Zapién Campos


#### Rank-abundance

- Avolio, M. L., Pierre, K. J. L., Houseman, G. R., Koerner, S. E., Grman, E., Isbell, F., ... & Wilcox, K. R. (2015). A framework for quantifying the magnitude and variability of community responses to global change drivers. Ecosphere, 6(12), 1-14.
- Erez, A., Lopez, J. G., Meir, Y., & Wingreen, N. S. (2021). Enzyme regulation and mutation in a model serial-dilution ecosystem. Physical Review E, 104(4), 044412.
- Mac Nally, R. (2007). Use of the abundance spectrum and relative-abundance distributions to analyze assemblage change in massively altered landscapes. The American Naturalist, 170(3), 319-330.
- Yoshino, Y., Galla, T., & Tokita, K. (2008). Rank abundance relations in evolutionary dynamics of random replicators. Physical Review E, 78(3), 031924.
- Zhang, L., Chen, L., Yu, X., Duvallet, C., Isazadeh, S., Dai, C., ... & Ling, F. (2020). Microbial Species Abundance Distributions Guide Human Population Size Estimation from Sewage Microbiomes. bioRxiv, 2020-12.
- Ho, P. Y., Good, B. H., & Huang, K. C. (2022). Competition for fluctuating resources reproduces statistics of species abundance over time across wide-ranging microbiotas. Elife, 11, e75168.

#### Criticality and Phase-Transitions

- Proverbio, D., Montanari, A. N., Skupin, A., & Gonçalves, J. (2022). Buffering variability in cell regulation motifs close to criticality. Physical Review E, 106(3), L032402.
- Arnold, J., & Schäfer, F. (2022). Replacing neural networks by optimal analytical predictors for the detection of phase transitions. Physical Review X, 12(3), 031044.
- Hu, J., Amor, D. R., Barbier, M., Bunin, G., & Gore, J. (2022). Emergent phases of ecological diversity and dynamics mapped in microcosms. Science, 378(6615), 85-89.
- Matthias Huelsmann, Martin Ackermann, Community instability in the microbial world, Science, 378, 6615, (29-30), (2022).
/doi/10.1126/science.ade2516
- Nazarimehr, F., Jafari, S., Perc, M., & Sprott, J. C. (2020). Critical slowing down indicators. Europhysics Letters, 132(1), 18001.
- Dakos, V., Van Nes, E. H., d'Odorico, P., & Scheffer, M. (2012). Robustness of variance and autocorrelation as indicators of critical slowing down. Ecology, 93(2), 264-271.
- 


#### Co-existence

- Jia, X., Dini-Andreote, F. & Salles, J.F. Unravelling the interplay of ecological processes structuring the bacterial rare biosphere. ISME COMMUN. 2, 96 (2022). https://doi.org/10.1038/s43705-022-00177-6
- Hanusch, M., He, X., Ruiz-Hernández, V. et al. Succession comprises a sequence of threshold-induced community assembly processes towards multidiversity. Commun Biol 5, 424 (2022). https://doi.org/10.1038/s42003-022-03372-2
- Emergent coexistence in multispecies microbial communities Chang, C-Y., Bajic, D., Vila, J.C.C., Estrela, S. & Sanchez A. BioRxiv (2022) https://www.biorxiv.org/content/10.1101/2022.05.20.492860v2.full.pdf


#### Networks

- Monica Steffi Matchado, Michael Lauber, Sandra Reitmeier, Tim Kacprowski, Jan Baumbach, Dirk Haller, Markus List,
Network analysis methods for studying microbial communities: A mini review,
Computational and Structural Biotechnology Journal,Volume 19, 2021,Pages 2687-2698,ISSN 2001-0370, https://doi.org/10.1016/j.csbj.2021.05.001.
- Raulo, A., Allen, B.E., Troitsky, T. et al. Social networks strongly predict the gut microbiota of wild mice. ISME J 15, 2601–2613 (2021). https://doi.org/10.1038/s41396-021-00949-3
- Zhaoqian Liu, Anjun Ma, Ewy Mathé, Marlena Merling, Qin Ma, Bingqiang Liu, Network analyses in microbiome based on high-throughput multi-omics data, Briefings in Bioinformatics, Volume 22, Issue 2, March 2021, Pages 1639–1655, https://doi.org/10.1093/bib/bbaa005
- Cao, X., Dong, A., Kang, G., Wang, X., Duan, L., Hou, H., ... & Wu, R. (2022). Modeling spatial interaction networks of the gut microbiota. Gut microbes, 14(1), 2106103.
- Lugo-Martinez, J., Ruiz-Perez, D., Narasimhan, G., & Bar-Joseph, Z. (2019). Dynamic interaction network inference from longitudinal microbiome data. Microbiome, 7, 1-14.
- Mainali K, Bewick S, Vecchio-Pagan B, Karig D, Fagan WF (2019) Detecting interaction networks in the human microbiome with conditional Granger causality. PLOS Computational Biology 15(5): e1007037. https://doi.org/10.1371/journal.pcbi.1007037
- Lim, R., Cabatbat, J. J. T., Martin, T. L., Kim, H., Kim, S., Sung, J., ... & Kim, P. J. (2020). Large-scale metabolic interaction network of the mouse and human gut microbiota. Scientific Data, 7(1), 204.
- Mehdi Layeghifard, David M. Hwang, David S. Guttman,
Disentangling Interactions in the Microbiome: A Network Perspective,Trends in Microbiology,Volume 25, Issue 3,2017,Pages 217-228,ISSN 0966-842X,https://doi.org/10.1016/j.tim.2016.11.008.
- Mehdi Layeghifard, David M. Hwang, David S. Guttman,Disentangling Interactions in the Microbiome: A Network Perspective,Trends in Microbiology,Volume 25,Issue 3,2017,Pages 217-228,ISSN 0966-842X,https://doi.org/10.1016/j.tim.2016.11.008.
- Wen, T., Xie, P., Yang, S., Niu, G., Liu, X., Ding, Z., ... & Yuan, J. (2022). ggClusterNet: An R package for microbiome network analysis and modularity‐based multiple network layouts. iMeta, 1(3), e32.

#### Community Assembly

-  Assembly of gut-derived bacterial communities follows early-bird resource utilization dynamics
Aranda-Diaz, A., Willis L., Nguyen T., Ho, P-Y., Vila, J.C.C., Thomsen T., Chavez, T., Yan, J., Yu, F. B., Neff, N., Sanchez A, Estrela S.# Huang K.C.# BioRxiv (2023) https://www.biorxiv.org/content/10.1101/2023.01.13.523996v1
- Predictability of the community-function landscape in wine yeast ecosystems, Ruiz, J., De Celis, M., Diaz-Colunga, J., Vila J.C.C., Benitez-Dominguez, B., Vicente J., Santos, A., Sanchez A., & Belda, I. BioRxiv (2022) https://doi.org/10.1101/2022.12.15.520418
-  Environmental modulation of global epistasis is governed by effective genetic interactions Diaz-Colunga, J.#, Sanchez A., & Ogbunugafor, C.B., BioRxiv (2022) https://www.biorxiv.org/content/10.1101/2022.11.02.514859v1.full.pdf
-  



#### Disbiosis

#### Other

- Juang, D. S., Wightman, W. E., Lozano, G. L., Barkal, L. J., Yu, J., Garavito, M. F., ... & Beebe, D. J. (2022). Microbial Community Interactions on a Chip. bioRxiv, 2022-10.
